common/unsafe/src/test/java/org/apache/spark/unsafe/types/CalendarIntervalSuite.java

```diff
@@ -274,4 +249,23 @@ private static void testSingleUnit(String unit, int number, int months, long mic
     assertEquals(fromString(input1), result);
     assertEquals(fromString(input2), result);
   }
+
+  @Test
+  public void fromStringCaseSensitivityTest() {
```

Comment:
This only have `positive` test cases.

Reply:
What kind of negative test do you want to see?

Reply:
Since you recovered the previous test case removal, it's okay for now.